target/linux/imx6/image/lumi.mk

```diff
@@ -0,0 +1,33 @@
+#
+
+define Device/xiaomi_gateway-lumi
+  DEVICE_VENDOR := Xiaomi
+  DEVICE_MODEL := Lumi Gateway
+  BOARD_NAME := xiaomi,gateway-lumi
+  DEVICE_PACKAGES := kmod-bluetooth \
+  	kmod-sound-core kmod-button-hotplug kmod-bluetooth \
+  	bluez-daemon bluez-libs bluez-utils alsa-utils hostapd \
+  	wpa-supplicant ca-certificates kmod-gpio-button-hotplug \
+  	kmod-i2c-core kmod-iio-core nand-utils
+
+  KERNEL_INSTALL := 0
+  KERNEL_NAME := zImage
+  KERNEL_SUFFIX := -zImage
+  KERNEL := kernel-bin
+  IMAGES := sysupgrade.bin dtb rootfs.bin
+  IMAGE/sysupgrade.bin := sysupgrade-tar | append-metadata
+  IMAGE/rootfs.bin := append-rootfs
+  IMAGE/dtb := install-dtb
+
+  DEVICE_DTS := \
+	imx6ull-xiaomi-lumi
+endef
+
+
+define Device/xiaomi_dgnwg05lm
+  $(Device/xiaomi_gateway-lumi)
+  DEVICE_MODEL := DGNWG05LM (RTL8723BS wifi)
```

Comment:
Is the WiFi relevant here (i.e. are there multiple devices with the same name, but different Wifi)?

Reply:
There is Aqara ZHWG11LM (RTL8189ES wifi) with the same hardware. I want to add one more device 
Aqara ZHWG11LM (RTL8189ES wifi)

Do you think I should remove the extra mentioning of the wifi chip?

Reply:
Mentioning the Wifi chip in the device name would only be necessary if the devices would otherwise have the exact same name (i.e could not be distinguished otherwise).

Reply:
Fixed this in the new patch